sdks/RepoToolset/tools/StrongName.targets

```diff
@@ -50,6 +50,11 @@
     </When>
   </Choose>
 
+  <!-- Full signing is not currently supported on non-Windows -->
```

Comment:
I believe the compilers already support it. Shall we move to the newer compilers version instead?

Reply:
Some reps aren't using the tool set compilers and are using a particular cli build instead.

Reply:
Not in all cases. For something like release/2.0.0, we are explicitly building with a 2.0 build of the cli.

Reply:
Then I'd say the repo should set this. We want to use the latest compilers and features by default.

Reply:
I agree.

IMHO: The lag of toolset adoption means that either RepoToolset or individual repos need to tweak certain settings. That will be true for this change and others in the future. I would rather make it more painful for repos who are not adopting completely compatible toolsets than add too much complexity to the central tools.